File: crates/target_csharp_system_text/output/initialisms/RootNestedIdInitialism.cs

using System.Text.Json.Serialization;

namespace JtdCodegenE2E
{

    public class RootNestedIdInitialism
    {


        [JsonPropertyName("json")]
        
        public string Json { get; set; }


        [JsonPropertyName("normalword")]
        
        public string Normalword { get; set; }

    }
}

File: crates/target_java_jackson/output/initialisms/RootNestedIdInitialism.java
package com.example;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;

@JsonSerialize
public class RootNestedIdInitialism {
    @JsonProperty("json")
    private String json;

    @JsonProperty("normalword")
    private String normalword;

    public RootNestedIdInitialism() {
    }

    /**
     * Getter for json.<p>
     */
    public String getJson() {
        return json;
    }

    /**
     * Setter for json.<p>
     */
    public void setJson(String json) {
        this.json = json;
    }

    /**
     * Getter for normalword.<p>
     */
    public String getNormalword() {
        return normalword;
    }

    /**
     * Setter for normalword.<p>
     */
    public void setNormalword(String normalword) {
        this.normalword = normalword;
    }
}
